docs/user/reuse_garage_policy.md

```diff
@@ -0,0 +1,54 @@
+# Load and Use a Trained Policy
+
+In this section you will learn how to save and load snapshots of your experiment
+while training. We will also show you how to extract a trained policy from a
+snapshot.
+
+## Saving an experiment snapshot
+
+Experiment activities are governed by a `LocalRunner` or a `LocalTFRunner`.
+Once setup, each of these classes has a `save(epoch)` method. This method
+pickles everything necessary to reconstruct your experiment, such as:
+
+- The runner's `setup_args` and `train_args`
+  - Random seed
+  - Batch size
+  - Number of epochs
+  - And [more](https://github.com/rlworkgroup/garage/blob/175ac4c90a408e2314d91cdbe95e419b183e0684/src/garage/experiment/local_runner.py#L393)
+- The experiment's `stats`
+- The environment
+- The algorithm (which includes the policy)
+- `n_workers`, `worker_class`, and `worker_args`
+
+To use the `save(epoch)`, setup an experiment as normal. Call the save
+method at the end. Example experiments can be found in the `examples`
+folder.
+
+By default, the data is saved to `./data/local/experiment`, but you can
+specify your own directory by setting `snapshot_dir` when instantiating
+the `LocalRunner`.
+
+## Extracting a trained policy from a snapshot
+
+If instead you wish to extract the trained policy from a saved experiment,
+the process is even simpler. You can do it in a few lines like this:
+
+```python
+# Load the policy
+from garage.experiment import SnapshotConfig, Snapshotter
+snapshotter = Snapshotter()
+data = snapshotter.load('path/to/snapshot/dir')
+policy = data['algo'].policy
```

Comment:
this code only extracts the policy -- it doesn't make use of it.

can you show the user how to run the policy in an environment?